flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is really unfortunate

Reply:
I'd rather wait until it gets Tier 1, but will think about it!

(https://crystal-lang.org/reference/1.3/platform_support.html#tier-2)
> Tier 2 platforms can be thought of as “expected to build”. Automated tests are not run so it’s not guaranteed to produce a working build, but platforms often work to quite a good degree and patches are always welcome!